lib/chore/consumer.rb

```diff
@@ -21,11 +21,6 @@ def initialize(queue_name, opts={})
     def self.reset_connection!
     end
 
-    # Cleans up any resources that were left behind from prior instances of the
-    # chore process.  By default, this is a no-op.
-    def self.cleanup(queue)
-    end
-
```

Comment:
just curious, why are you removing this from the "base class"?
i get that we are now providing an implementation in filesystem consumer, but question maybe still relevant?

Reply:
oh, think i got it now, nevermind